File: Part1/calculate_sum_errors.py
# calculate_sum_errors.py
# Errors introduced: missing colons, bad function call

def calculate_sum(arr)             # SyntaxError: expected ':'
    total = 0
    for num in arr                # SyntaxError: expected ':'
        total += num
    return total

print( calculate sum([1, 2, 3, 4, 5]) )  # SyntaxError or NameError

File: Part1/calculateSumErrors.js
// calculateSumErrors.js
// Errors: bad parenthesis, misspelled call, missing brace

function calculateSum(arr {           
    let total = 0;
    for (let num of arr) {
      total += num;
    }
    return total;
    // missing closing brace
  }
  
  let result = calcualteSum([1, 2, 3, 4, 5]);  // ReferenceError
  console.log("Sum:", result);
  